src/components/HomeView.vue

```diff
@@ -30,19 +48,59 @@ export default {
     if (!this.loggedIn) {
       this.$router.push("/login");
     }
-    // UserService.getPublicContent().then(
-    //   (response) => {
-    //     this.content = response.data;
-    //   },
-    //   (error) => {
-    //     this.content =
-    //       (error.response &&
-    //         error.response.data &&
-    //         error.response.data.message) ||
-    //       error.message ||
-    //       error.toString();
-    //   }
-    // );
   },
 };
 </script>
+<!-- TODO(#16): fix css to keep button active when clicked off, it was until it suddenly stopped -->
+<!-- TODO(#16): set script to have the inital tab (IngredientFeed) button active -->
+<!-- TODO(#17): remove App.vue routing and routing in router.js for everything other than home-->
```

Comment:
I would prefer you did this in this PR please; you can just remove the stuff from router.js and App.vue that links to recipes, ingredients, and shopping list and then delete this TODO